db/types/categories.py

```diff
@@ -72,6 +71,7 @@
 NUMERIC_TYPES = frozenset({
     *INTEGER_TYPES,
     *DECIMAL_TYPES,
+    PostgresType.NUMERIC
```

Comment:
Why this change? `NUMERIC` is a decimal type.

Reply:
My intention was to exclude it from being used in the `_get_decimal_number_type_body_map` function which creates 'default' casting function for other decimal types but to still treat it as a numeric type.